google-cloud-clients/google-cloud-redis/src/test/java/com/google/cloud/redis/it/ITSystemTest.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.redis.it;
+
+import static org.junit.Assert.assertEquals;
+
+import com.google.cloud.ServiceOptions;
+import com.google.cloud.redis.v1beta1.CloudRedisClient;
+import com.google.cloud.redis.v1beta1.Instance;
+import com.google.cloud.redis.v1beta1.InstanceName;
+import com.google.cloud.redis.v1beta1.LocationName;
+import com.google.common.collect.Lists;
+import java.util.List;
+import java.util.UUID;
+import java.util.logging.Logger;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ITSystemTest {
+
+  private static CloudRedisClient client;
+  private static String projectId;
+
+  private static final Logger log = Logger.getLogger(ITSystemTest.class.getName());
+  private static final Instance.Tier TIER = Instance.Tier.BASIC;
+  private static final String INSTANCE_NAME_PREFIX = "test-instance";
+  private static final String INSTANCE =
+      INSTANCE_NAME_PREFIX + "-" + UUID.randomUUID().toString().substring(0, 8);
+  private static final String LOCATION = "us-central1";
+  private static final int MEMORY_SIZE_GB = 1;
+
+  @BeforeClass
+  public static void beforeClass() throws Exception {
+    client = CloudRedisClient.create();
+    projectId = ServiceOptions.getDefaultProjectId();
+
+    /** Creates a Redis instance based on the specified tier and memory size. */
+    LocationName parent = LocationName.of(projectId, LOCATION);
+    Instance instance = Instance.newBuilder().setTier(TIER).setMemorySizeGb(MEMORY_SIZE_GB).build();
+    client.createInstanceAsync(parent, INSTANCE, instance).get();
```

Comment:
This is failing with an invalid config: `Invalid authorized network 'default'. Legacy networks are not supported.`

Reply:
This is likely because our test project is very old and has some older grandfathered in settings (like default networking) still configured. I'll investigate some more.

Reply:
I'm hesitant to change the default network for the test project, so I created a new `redis-vpc` network in our test project. Let's make the test configurable (via a maven param like `-Dredis.network=redis-vpc` which we can add [here](https://github.com/googleapis/google-cloud-java/blob/cd38245528822af3a321fcd0a2edac4352f1c3e9/.kokoro/presubmit/redis-it.cfg#L11) (and in the continuous integration config). We can default that value to `default` for most other users who run the tests.

Reply:
sounds good. Thanks @chingor13